docs/docs/tutorial/part-7/index.mdx

```diff
@@ -0,0 +1,492 @@
+    * For example, a post at `blog/my-first-post.mdx` would move to `blog/my-first-post/index.mdx`. Similarly, a post at `blog/another-post.mdx` would move to `blog/another-post/index.mdx`.
+
+<Announcement style={{marginBottom: "1.5rem"}}>
+
+**Note:** After you move or rename your `.mdx` files, you'll need to stop and restart your local development server for your changes to be picked up.
+
+</Announcement>
+
+![Diagram of moving each MDX file into its own subdirectory in the blog folder](./reorganize-blog-directory.png)
+
+2. Use a website like [Unsplash](https://unsplash.com) to find some pretty, freely usable images. For best results, choose photos with a landscape (horizontal) orientation, since those will fit on your screen more easily.
+    ![A screenshot of the Unsplash home page](./unsplash.png)
+
+2. When you've found a photo that you like, download it and add it to subdirectory for one of your blog posts. Continue downloading photos until you have a different hero image for each post.
+
+3. Next, add some additional frontmatter fields to each of your blog posts:
+    * `hero_image`: the relative path to the hero image file for that post
+    * `hero_image_alt`: a short description of the image, to be used as alternative text for screen readers or in case the image doesn't load correctly
+    * `hero_image_credit_text`: the text to display to give the photographer credit for the hero image
+    * `hero_image_credit_link`: a link to the page where your hero image was downloaded from
+
+```mdx:title=blog/my-first-post/index.js
+---
+title: "My First Post"
+date: "2021-07-23"
+// highlight-start
+hero_image: "./christopher-ayme-ocZ-_Y7-Ptg-unsplash.jpg"
+hero_image_alt: "A gray pitbull relaxing on the sidewalk with its tongue hanging out"
+hero_image_credit_text: "Christopher Ayme"
+hero_image_credit_link: "https://unsplash.com/photos/ocZ-_Y7-Ptg"
+// highlight-end
+---
+
+...
+```
+
+```mdx:title=blog/another-post/index.js
+---
+title: "Another Post"
+date: "2021-07-24"
+// highlight-start
+hero_image: "./anthony-duran-eLUBGqKGdE4-unsplash.jpg"
+hero_image_alt: "A grey and white pitbull wading happily in a pool"
+hero_image_credit_text: "Anthony Duran"
+hero_image_credit_link: "https://unsplash.com/photos/eLUBGqKGdE4"
+// highlight-end
+---
+
+...
+```
+
+```mdx:title=blog/yet-another-post/index.js
+---
+title: "Yet Another Post"
+date: "2021-07-25"
+// highlight-start
+hero_image: "./jane-almon-7rriIaBH6JY-unsplash.jpg"
+hero_image_alt: "A white pitbull wearing big googly-eye glasses"
+hero_image_credit_text: "Jane Almon"
+hero_image_credit_link: "https://unsplash.com/photos/7rriIaBH6JY"
+// highlight-end
+---
+
+...
+```
+
+
+Now that your hero images are set up, it's time to connect them to the data layer so you can pull them into your blog post page template.
```

Comment:
A couple of sentences begin with "Now that" so potential for some variation here

Reply:
Good call. Made some edits!